homeassistant/components/lcn/__init__.py

```diff
@@ -252,14 +249,15 @@ class LcnEntity(Entity):
     def __init__(
         self,
         config: ConfigType,
-        config_entry: config_entries.ConfigEntry,
+        config_entry: ConfigEntry,
     ) -> None:
         """Initialize the LCN device."""
         self.config = config
         self.config_entry = config_entry
-        self.address: AddressType = self.config[CONF_ADDRESS]
+        self.address: AddressType = config[CONF_ADDRESS]
         self._unregister_for_inputs: Callable | None = None
         self._name: str = config[CONF_NAME]
+        self.device_connection: DeviceConnectionType | None = None
```

Comment:
remove this outside of the constructor
```suggestion
       device_connection: DeviceConnectionType
```

Reply:
Well, then I get this pylint warning that an attribute was defined outside __init__.

Reply:
I don't think so? Can you try committing that?

Also, can we move the base entity to entity.py?